front-end/src/index.js

```diff
@@ -0,0 +1,10 @@
+import { createRoot } from 'react-dom/client';
```

Comment:
this repo does not put spaces after { or before }. the main repos do, but this repo marches to a different drummer. it makes little sense to me - we don't use them for  `(` or for `[`, so why `{`? the only argument is the debugger displays them that way, but from my pov it's just wasted space with no benefit.